crates/iota-core/src/congestion_tracker.rs

```diff
@@ -83,21 +111,16 @@ impl CongestionInfo {
 /// `CongestionTracker` tracks objects' congestion info.
 /// The info is then used to calculated a suggested gas price.
 pub struct CongestionTracker {
-    /// Key-value-based cache storing congestion info of objects.
-    object_congestion_info: Cache<ObjectID, CongestionInfo>,
-}
-
-impl Default for CongestionTracker {
-    fn default() -> Self {
-        Self::new()
-    }
+    pub reference_gas_price: u64,
+    pub object_congestion_info: Cache<ObjectID, CongestionInfo>,
```

Comment:
```suggestion
    reference_gas_price: u64,
    object_congestion_info: Cache<ObjectID, CongestionInfo>,
```
Used only in this module, so does not need to be public.